x-pack/plugins/spaces/server/lib/utils/url.test.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+// TEMPORARY UNTIL FIXED!
+// DIRECT COPY FROM `src/core/utils/url`, since it's not possible to import from there,
```

Comment:
does it fail on CI?

Reply:
Yeah, last time I checked, I'm not able to import from `src/core/utils` into xpack. Even re-exporting this from 'src/core/server` didn't work for me.